oc_cdtapi/tests/test_BitbucketAPI.py

```
@@ -0,0 +1,263 @@
+class _BitbucketAPI(BitbucketAPI):
+
+    def __init__(self, *args, **argv):
+        super(_BitbucketAPI, self).__init__(*args, **argv)
+        self.web = _Session(self._read_url)
+
+    def _read_url(self, url, method='GET', data=None):
+        """
+        Override http request method to return fake responses
+        """
+        # GET /rest/api/1.0/projects/PROJ/repos/my-repo
+        if re.match('.+\/projects\/.+\/repos\/[^\/]+$', url) and method == 'GET':
```

Ruff (0.15.12)

[warning] Invalid escape sequence: `\/`

Use a raw string literal

(W605)